parent/pom.xml

```diff
@@ -41,10 +41,8 @@
         <ignite-spring-boot-autoconfigure-ext.version>1.1.0-SNAPSHOT</ignite-spring-boot-autoconfigure-ext.version>
         <ignite-spring-boot-thin-client-autoconfigure-ext.version>1.1.0-SNAPSHOT</ignite-spring-boot-thin-client-autoconfigure-ext.version>
         <ignite-spring-cache-ext.version>1.1.0-SNAPSHOT</ignite-spring-cache-ext.version>
-        <ignite-spring-data-2.0-ext.version>1.1.0-SNAPSHOT</ignite-spring-data-2.0-ext.version>
-        <ignite-spring-data-2.2-ext.version>1.1.0-SNAPSHOT</ignite-spring-data-2.2-ext.version>
-        <ignite-spring-data-commons.version>1.2.0-SNAPSHOT</ignite-spring-data-commons.version>
         <ignite-spring-data-ext.version>1.1.0-SNAPSHOT</ignite-spring-data-ext.version>
```

Comment:
I think this is not a minor fix and the new version should be 2.X.X.

Reply:
@NSAmelchev Do you suggest raising versions of just `ignite-spring-data-ext` and `ignite-spring-data-commons`, or all 'spring-related' extensions (`spring-boot-autoconfigure-ext`, `spring-boot-thin-client-autoconfigure-ext`, `spring-cache-ext`, `spring-tx-ext`)?

Reply:
@rpuch , only for related `spring-data-ext`. Extensions are independent modules.

Reply:
Changed versions of ignite-spring-data-ext and ignite-spring-data-commons to 2.0.0.